source/codegen/templates/service.cpp.mako

```diff
@@ -37,6 +37,14 @@ namespace ${config["namespace_component"]}_grpc {
   {
   }
 
+% if common_helpers.has_viboolean_array_param(functions):
+  void ${service_class_prefix}Service::Copy(const std::vector<ViBoolean>& input, google::protobuf::RepeatedField<bool>* output) 
+  {
+    for (auto item : input) {
+      output->Add(item != 0);
```

Comment:
Instead of `item != 0`, I guess I can just do `(bool)item` which seems to compile fine. But I am not sure if it is ok to typecast it like that.

Reply:
Also I can't just do `output->Add(item)`, I guess since ViBoolean is 2 bytes (unsigned short) and bool is 1 byte, doing `output->Add(item)` adds multiple objects to `output` array.

Reply:
```suggestion
      output->Add(item != VI_FALSE);
```

Reply:
Changed to VI_FALSE.